### PR TITLE
add a networking constants class with the metaverse server URL

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -138,8 +139,8 @@
 
 // ON WIndows PC, NVidia Optimus laptop, we want to enable NVIDIA GPU
 #if defined(Q_OS_WIN)
-extern "C" {
- _declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
+extern "C" {
+ _declspec(dllexport) DWORD NvOptimusEnablement = 0x00000001;
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: ??

Comment (birarda) [inline interface/src/Application.cpp]: weird - not intentional ... is there a diff between the two lines?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 61806935bb3118a47b04c2572b7b183a6253d6f8 available at: https://gist.github.com/d11f8f0d2cdf92377ea7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4559.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4559.dmg

File: interface/src/Application.cpp (hunk 1 above)
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: In upstream/master those two lines have DOS line endings.  I assume that your editor correctly removed them for you.

Comment (AndrewMeadows) [inline interface/src/Application.cpp]: Blame tool says @samcake added those line endings.  One of his tools is not configured correctly.  Perhaps it is happening during a git merge?  Dunno.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e22b33e96dd8e8c342d6d3e5f6822207f5cb236d available at: https://gist.github.com/e4d80ba4f47f3670b4e3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4559.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4559.dmg

**Pull request merged**